src/Hud/Loot/ItemAlertPlugin.cs

```diff
@@ -129,7 +129,7 @@ public override void Render()
                         }
                     }
                 }
-                var currentAlertsArray = currentAlerts.Where(x => x.Key != null && x.Key.Address != 0 && x.Key.IsValid).ToArray();
+                var currentAlertsArray = currentAlerts.ToList().Where(x => x.Key != null && x.Key.Address != 0 && x.Key.IsValid).ToArray();
```

Comment:
why convert it to ToList()?

Reply:
nevermind found the reason.

Reply:
https://github.com/TehCheat/PoEHUD/commit/ab42013b676054e391d0f4456f2aef51c81e3d72